db/migrations/005.do.retrieval-result-status.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE daily_retrieval_result_status (
+  day DATE NOT NULL,
+  status TEXT NOT NULL,
+  rate NUMERIC NOT NULL,
+  PRIMARY KEY (day, status)
+);
```

Comment:
Since you are going to query this table using a `day` range, please add an index to speed up those queries.

Reply:
caf8af7